cpp/ql/src/Security/CWE/CWE-311/CleartextTransmission.ql

```diff
@@ -261,8 +261,7 @@ class ToEncryptionConfiguration extends TaintTracking2::Configuration {
   ToEncryptionConfiguration() { this = "ToEncryptionConfiguration" }
 
   override predicate isSource(DataFlow::Node source) {
-    any(FromSensitiveConfiguration config).hasFlow(source, _) and
-    isSourceImpl(source)
```

Comment:
Drive-by fix. This condition follows from `any(FromSensitiveConfiguration config).hasFlow(source, _)`.